### PR TITLE
Selfie: Add gallery preview

### PR DESCRIPTION
**Pull request by Dhina17**
- ~Need to add the correct thumbnail size~
- ~Make it circular~

### CODE REVIEW

**Comment by Dhina17** [comment]
#9 - Gallery Preview

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt [hunk 1]
```diff
@@ -11,14 +11,14 @@ import android.animation.ObjectAnimator
...
 import android.content.pm.PackageManager
+import android.graphics.Bitmap
 import android.graphics.Color
 import android.graphics.drawable.ColorDrawable
-import android.os.Bundle
-import android.os.Handler
-import android.os.Looper
-import android.os.Message
+import android.net.Uri
+import android.os.*
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/selfie/MainActivity.kt]: image
Also, do 
image

Comment (luk1337) [reply app/src/main/java/org/lineageos/selfie/MainActivity.kt]: Redo your imports after doing that ^^

Comment (Dhina17) [reply app/src/main/java/org/lineageos/selfie/MainActivity.kt]: done

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt [hunk 2]
```diff
@@ -696,13 +701,13 @@ class MainActivity : AppCompatActivity() {
...
     }
 
-    private fun updateGalleryIcon(uriExists: Boolean) {
-        if (uriExists) {
-            galleryButton.clearColorFilter()
-            galleryButton.setColorFilter(getColor(R.color.grey))
-        } else {
-            galleryButton.clearColorFilter()
+    private fun updateGalleryIcon(uri: Uri?) {
+        galleryButton.clearColorFilter()
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/selfie/MainActivity.kt:700]:
```
getThumbnail(uri)?.let {
    galleryButton.setImageBitmap(it)
} ?: run {
    galleryButton.setColorFilter(getColor(R.color.dark_grey))
}
```

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt [hunk 3]
```diff
@@ -760,6 +765,21 @@ class MainActivity : AppCompatActivity() {
         windowInsetsController.hide(WindowInsetsCompat.Type.systemBars())
     }
 
+    private fun getThumbnail(uri: Uri?): Bitmap? {
+
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/selfie/MainActivity.kt]: unnecessary empty line

File: app/src/main/java/org/lineageos/selfie/utils/ConvertUtils.kt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package org.lineageos.selfie.utils
+
+import android.content.Context
+import kotlin.math.roundToInt
+
+object ConvertUtils {
+    fun convertDptoPixel(context: Context, dp: Int): Int {
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/selfie/utils/ConvertUtils.kt]: dpToPx?

Comment (Dhina17) [reply app/src/main/java/org/lineageos/selfie/utils/ConvertUtils.kt]: done

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt (hunk 3 above)
Comment (Dhina17) [reply app/src/main/java/org/lineageos/selfie/MainActivity.kt]: done

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt (hunk 2 above)
Comment (Dhina17) [reply app/src/main/java/org/lineageos/selfie/MainActivity.kt:700]: done

**Comment by Dhina17** [comment]
@luk1337 Done.

**Comment by luk1337** [comment]
Not sure about using CardView, but this works ^.^

**Comment by SebaUbuntu** [comment]
lgtm too

**Pull request merged**